__tests__/app.test.js

```diff
@@ -53,4 +55,54 @@ describe('GET /api/categories',()=>{
             expect(res.body.msg).toEqual('Route not found')
         })
     })
-})
\ No newline at end of file
+})
+
+describe('GET /api/reviews', ()=>{
+    test('responds with staus code 200', ()=>{
+        return request(app)
+        .get('/api/reviews')
+        .expect(200)
+    })
+    test('responds with an array of review objects', ()=>{
+        return request(app)
+        .get('/api/reviews')
+        .expect(200)
+        .then((res)=>{
+            expect(Array.isArray(res.body.reviews)).toBe(true)
+            res.body.reviews.forEach((review)=>{
+                expect(typeof review).toBe('object')
+                expect(review).toEqual(
+                    expect.objectContaining({
+                        owner: expect.any(String),
+                        title: expect.any(String),
+                        review_id: expect.any(Number),
+                        category: expect.any(String),
+                        review_img_url: expect.any(String),
+                        created_at: expect.any(String),
+                        votes: expect.any(Number),
+                        designer: expect.any(String),
+                        comment_count: expect.any(String)
+                    })
+                )
+            })
+        })
+    })
+    test('Reviews are ordered in descending order of dates', ()=>{
+        return request(app)
+        .get('/api/reviews')
+        .expect(200)
+        .then((res)=>{
+            expect(res.body.reviews).toBeSortedBy('created_at', {descending: true, coerce: true})
+        })
+    })
+    test('responds with route not found when api address spelt wrong', ()=>{
+        return request(app)
+        .get('/api/review')
+        .expect(404)
+        .then((res)=>{
+            expect(res.body.msg).toEqual('Route not found')
+        })
```

Comment:
You don't need to do the above for every endpoint and actually it is not necessarily linked to this endpoint at all. I would just have a similar test once and it should be in it's own describe block